WordPress/Classes/ViewRelated/Post/Prepublishing/PrepublishingSocialAccountsViewController.swift

```diff
@@ -203,18 +235,28 @@ private extension PrepublishingSocialAccountsViewController {
         return cell
     }
 
-    func updateConnection(at index: Int, enabled: Bool) {
-        guard index < connections.count else {
+    func valueForConnection(at index: Int) -> Bool {
+        guard let connection = connections[safe: index] else {
+            return false
+        }
+        return connectionChanges[index] ?? connection.isOn
```

Comment:
In `updateConnection`, `connectionChanges` is accessed with `connection.keyringID`. Should it be the same here?

```suggestion
        return connectionChanges[connection.keyringID] ?? connection.isOn
```

Reply:
Thanks for catching this! I've fixed this in https://github.com/wordpress-mobile/WordPress-iOS/pull/21188/commits/e36337e1e9a02f6865290bd31180be3574922a0f.